airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -97,7 +98,7 @@ static String getDatasource(final String dataset, final String connector) throws
     log.info("datasourceFilename {}", datasourceFilename);
     try (final var reader =
         new BufferedReader(new InputStreamReader(Objects.requireNonNull(
-            Thread.currentThread().getContextClassLoader().getResourceAsStream(datasourceFilename))))) {
+            Thread.currentThread().getContextClassLoader().getResourceAsStream(datasourceFilename)), StandardCharsets.UTF_8))) {
```

Comment:
😆  I was just about to push this change up too, thanks for fixing this first

Reply:
great minds, eh?